docs/Installation/InstallationOnClusters.md

```diff
@@ -79,19 +79,3 @@ of `wheeler_gcc.sh`
 
 Follow the general instructions, using `ocean` for `SYSTEM_TO_RUN_ON`,
 you do not need to install any dependencies, so you can skip steps 5 and 6.
-
```

Comment:
fix commit message (you removed zwicky, not wheeler)

Reply:
Oops! Thanks for catching this!